examples/trainer/cleanlab/cleanlab_training_tutorial.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import os
+import sys
+from itertools import product
+
+from torch import Tensor, LongTensor
+from torch.nn import CrossEntropyLoss
+from torch.optim import Adam
+from torch.utils.data import TensorDataset
+
+from examples.trainer.dscrossweigh.dscrossweigh_training_tutorial import get_tfidf_features
+from examples.utils import read_train_dev_test
+from knodle.model.logistic_regression_model import LogisticRegressionModel
+from knodle.trainer.cleanlab.cleanlab import CleanLabTrainer
+from knodle.trainer.cleanlab.config import CleanLabConfig
+
+
+def train_cleanlab(path_to_data: str) -> None:
+    """ This is an example of launching cleanlab trainer """
+
+    parameters = dict(
+        seed=[12345], lr=[0.1], cv_n_folds=[3, 5, 8], prune_method=['prune_by_class', 'prune_by_noise_rate', 'both'],
+        epochs=[200], batch_size=[128], psx_calculation_method=['split_by_rules', 'random'],
+    )
+    parameter_values = [v for v in parameters.values()]
+
+    df_train, df_dev, df_test, train_rule_matches_z, _, mapping_rules_labels_t = read_train_dev_test(path_to_data)
+
+    train_input_x, test_input_x, _ = get_tfidf_features(df_train["sample"], test_data=df_test["sample"])
+
+    train_features_dataset = TensorDataset(Tensor(train_input_x.toarray()))
+    test_features_dataset = TensorDataset(Tensor(test_input_x.toarray()))
+
+    test_labels = list(df_test.iloc[:, -1])
+    test_labels_dataset = TensorDataset(LongTensor(list(df_test.iloc[:, 1])))
```

Comment:
but e.g. in tac dataset the column "label" contains labels as words, while the label indices (which should be used here) are in the last column called "enc_label". 

Another possible solution would be to rename the tac columns so that the column "label" would contain label ids, while the labels-words would be in another column (somehow "text labels"). Do you think it would be a better option?